Firestore/core/src/firebase/firestore/remote/stream.h

```diff
@@ -129,15 +129,15 @@ class Stream : public GrpcStreamObserver,
    *
    * When start returns, `IsStarted` will return true.
    */
-  void Start();
+  virtual /*only virtual for tests*/ void Start();
```

Comment:
Another approach would be to leave as non-virtual, and adjust the behaviour via the ctor params. (Take a look at go/totw/labs/virtual-for-testing for details.) Probably not worth it here; this is fine.

Reply:
The `/*only virtual for tests*/` decoration seems like overkill. Maybe just add a class-level comment about how this isn't intended to be subclassed generally but contains virtual methods for testing?

Reply:
Hmm, `Stream` is intended as a base class; anyway, made a class comment that all public virtual methods are for tests.